io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -211,11 +210,7 @@ impl<'a, 'b: 'a> AcceptFsm for &'a mut AcceptLink<'b> {
         };
 
         // Compute the minimum batch size
-        state.transport.batch_size = state
-            .transport
-            .batch_size
-            .min(init_syn.batch_size)
-            .min(batch_size::UNICAST);
+        state.transport.batch_size = state.transport.batch_size.min(init_syn.batch_size);
```

Comment:
I'd rather silent this warning. It's convenient to be able to change a constant in the protocol and having all the changes reflected in the overall transport.